commons/src/main/resources/measures-data.json

```diff
@@ -44044,7 +44044,42 @@
 			"electronicHealthRecord": "https://ecqi.healthit.gov/ecqm/measures/cms347v1"
 		},
 		"cpcPlusGroup": "Other_Measure",
-		"eMeasureUuid": "40280382-5b4d-eebc-015b-8245e0fa06b7"
+		"eMeasureUuid": "40280382-5b4d-eebc-015b-8245e0fa06b7",
```

Comment:
Will the measures-data changes also be made to the source version of the file?

Reply:
Yes. I have already made PR to change them. I had to verify with Sarah White that it wouldn't break anything on their end.